### PR TITLE
Some new questions added

### PR DESCRIPTION
**Pull request by rajat123456**

### CODE REVIEW

File: Week02/README.md [hunk 1]
```diff
@@ -36,10 +35,8 @@
 
 ## Long Arithmetic (Multi, Sum, Div, Sub)
 
-#### [Tutorial]()
-
-#### Other Study Sources
-
+#### Study Sources
```
Comment (BedirT) [inline Week02/README.md]: Please try not to change the structure, Tutorial - Other Study Sources, etc.

File: Week02/README.md [hunk 2]
```diff
@@ -64,3 +61,9 @@
 - [Extra Long Factorials](https://www.hackerrank.com/challenges/extra-long-factorials?h_r=internal-search)
 	- [Bedir's Solution](https://github.com/BedirT/AlgorithmsL/blob/master/Problems/HackerRank/Algorithms/Implementation/Extra%20Long%20Factorial.cpp)
 	- [Nadide's Solution](https://github.com/nadide/ACM-ICPC/blob/master/problems/hackerrank/extraLongFactorials.c)
+- [GCD of Large Number](https://www.codechef.com/problems/GCD2)
+	- [Editorial](https://www.geeksforgeeks.org/gcd-of-two-numbers-when-one-of-them-can-be-very-large-2/)
+- [Modular GCD](https://www.codechef.com/AUG18B/problems/GCDMOD)
+- [Number of Balanced Binary Trees of Height h](https://www.geeksforgeeks.org/count-balanced-binary-trees-height-h/)
```
Comment (BedirT) [inline Week02/README.md]: Remove this one, it is not a clear question.

File: Week05/README.md [hunk 3]
```diff
@@ -9,6 +9,12 @@
 - [GeeksforGeeks - Stack](https://www.geeksforgeeks.org/stack-data-structure-introduction-program/)
 - [HackerRank - Video](https://www.youtube.com/watch?v=wjI1WNcIntg)
 
+#### Questions
+- [Redundant Bracket Question along with Java Solution](https://github.com/rajat123456/Stacks-And-Queues/blob/master/Redundant%20Brackets%20Teaser.java)
```
Comment (BedirT) [inline Week05/README.md]: Please re-arrange the question, let it be separate from the answer

File: Week06/README.md [hunk 4]
```diff
@@ -15,6 +15,16 @@
 - [Print the Elements of a Linked List](https://www.hackerrank.com/challenges/print-the-elements-of-a-linked-list/problem)
 - [Delete a Node](https://www.hackerrank.com/challenges/delete-a-node-from-a-linked-list/problem)
 - [Merge two sorted linked lists](https://www.hackerrank.com/challenges/merge-two-sorted-linked-lists/problem)
+- [Reverse Nodes in k-Group](https://leetcode.com/problems/reverse-nodes-in-k-group/description/)
+        [Java Solution](https://github.com/rajat123456/General-Competitive-Programming-Questions/blob/master/LeetCode(Reverse%20Nodes%20in%20k-Group).java)
+- [Remove Duplicates from Sorted List](https://leetcode.com/problems/remove-duplicates-from-sorted-list/description/)
+        [Java Solution](https://github.com/rajat123456/General-Competitive-Programming-Questions/blob/master/LeetCode(Remove%20Duplicates%20from%20Sorted%20List).java)
```
Comment (BedirT) [inline Week06/README.md]: add '-' at the beginning, check the format pls.

- Question name
   - solution

File: Week06/README.md [hunk 5]
```diff
@@ -15,6 +15,16 @@
...
+- [Add_1_to_number_represented_in_linked_list](https://github.com/rajat123456/LinkedList-in-Java/blob/master/Add_1_to_number_represented_in_linked_list.java)
+- [Sort_a_linked_list_of_0s_1s_and_2s](https://github.com/rajat123456/LinkedList-in-Java/blob/master/Sort_a_linked_list_of_0s_1s_and_2s.java)
```
Comment (BedirT) [inline Week06/README.md]: No need of '_'

File: Week06/README.md [hunk 6]
```diff
@@ -15,6 +15,16 @@
...
+- Check Loop in Link List
+        -[Approach 1](https://github.com/rajat123456/LinkedList-in-Java/blob/master/CheckLoopInLinkList1.java)
```
Comment (BedirT) [inline Week06/README.md]: Same here

Space before '['

**Review by BedirT — CHANGES_REQUESTED** [review]
Minor mistakes about formatting, please fix and push again

File: Week02/README.md (hunk 2 above)
Comment (rajat123456) [reply Week02/README.md]: Which question is to remove ?

**Comment by BedirT** [comment]
Any updates ?

**Comment by rajat123456** [comment]
> Any updates ?

@BedirT  sir, yes i am waiting from last month for your reply that what question should be removed, in the section which you have written "Remove this "

File: Week02/README.md (hunk 2 above)
Comment (BedirT) [reply Week02/README.md]: Number of Balanced Binary Trees of height h

**Comment by BedirT** [comment]
@rajat123456 I just saw the question 😄 wasted a month for nothing then. Anyways 👍

**Pull request closed without merging**